### ISSUE TITLE
You get loot popups even if you didnt participate in a fight.

### ISSUE THREAD

**Issue by Cabro** [LightsHope/server#324]
Watched a warlock solo the first guard in DM north, and when it died i couldnt check the corpse for loot. - working OK
Then he looted the boss and i had loot popups. - NOT OK, i shouldn't have gotten those since i didnt participate in the fight.

**Comment by GuybrushGit** [comment]
Loot is determined by range.

**Comment by Cabro** [comment]
You are wrong here. How come i couldn't loot it? The corpse showed as having no loot for me.

**Comment by ifyousayso** [comment]
Not sure about this. I didn't play on retail before 2016, also not a hunter, but I always got full credit for all kills when drinking while party members killed stuff, as long as I was in range.

http://wowwiki.wikia.com/wiki/Party : "However, if a party member is out of range when a monster is killed.."
"out of range" is the implementation of the "and so did not contribute to the kill" design. It doesn't mean that the design had implementations in all scenarios. Therefore, if you're in range, you are contributing, even if not in combat. What if someone gets in combat but doesn't attack?.. or attacks and misses?.. or only heals themselves (building threat)?

https://eu.battle.net/forums/en/wow/topic/5847766569 : "That was not the case . In the old days hunter went into spots where mobs spawn fast and go afk and the pet killed mobs (giving em xp ) Blizzard fixed it and made that if you pet kills a mob without you hitting it you wont get xp or loot"
This has everything to do with pets and nothing to do with other players. I'm not saying this disproves your claim, just pointing to the words. A lot of fixes are exceptions to the rules because people were caught abusing them.

Also, if you couldn't loot it, I'm more likely to believe that is a bug here, 'cause you should've been able to.
